tests/ui/parser/ternary_operator.rs

```diff
@@ -0,0 +1,38 @@
+#![no_main]
```

Comment:
nit: there's no reason to use `no_main` here :sweat_smile: either add a dummy main, or use `// compile-flags: --crate-type=lib` or something. adding a dummy main is the easiest, tho.

Reply:
Yeah that's just generally how it's done in clippy ^^ I'm fine with anything else